clients/cloud/confluent-cli/confluent-cli-example.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Set topic name
+topic_name=test1
+
+# Create topic in Confluent Cloud
+ccloud topic create $topic_name
+
+# Produce messages
+num_messages=10
+(for ((i=0;i<$num_messages;i++)); do echo "alice,{\"count\":${i}}" ; done) | \
```

Comment:
Yeah, I mean you can use `for i in seq 0 $num_messages; do ..` to make it simpler. Anyway, doesn't matter

Reply:
Simplified a bit